Core/tests/Unit/LongRunning/OperationResponseTraitTest.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * Copyright 2020 Google Inc.
```

Comment:
```suggestion
 * Copyright 2020 Google LLC
```

Make sure to update any snippets to use the proper copyright. :)

Reply:
still not fixed.